service.subtitles.wizdom/service.py

```diff
@@ -1,297 +1,345 @@
-from os import path
+import sys,codecs
 from requests import get
+from os import path
 from json import loads, load
-from shutil import copyfileobj, rmtree
+from shutil import rmtree
 from time import time
 from unicodedata import normalize
-from urllib import urlretrieve, unquote_plus, unquote, urlopen, quote
+from urllib import unquote_plus, unquote, quote
+from urlparse import urlparse
 from xbmcaddon import Addon
 from xbmcplugin import endOfDirectory, addDirectoryItem
 from xbmcgui import ListItem, Dialog
 from xbmcvfs import listdir, exists, mkdirs
-from xbmc import translatePath, executebuiltin, getInfoLabel, executeJSONRPC, Player, sleep, log, getCondVisibility
-from re import sub
-import sys
-
-reload(sys)
-sys.setdefaultencoding('utf8')
-
-__addon__ = Addon()
-__scriptid__ = __addon__.getAddonInfo('id')
-__version__ = __addon__.getAddonInfo('version')
-__temp__ = unicode(translatePath(__addon__.getAddonInfo('profile')), 'utf-8')
-__subs__ = unicode(translatePath(path.join(__temp__, 'subs')), 'utf-8')
-__scriptname__ = __addon__.getAddonInfo('name')
-__language__ = __addon__.getLocalizedString
-
+from xbmc import translatePath, executebuiltin, getInfoLabel, executeJSONRPC, Player, log, getCondVisibility
+
+myAddon = Addon()
+myScriptID = myAddon.getAddonInfo('id')
+myVersion = myAddon.getAddonInfo('version')
+myTmp = translatePath(myAddon.getAddonInfo('profile')).encode('utf-8')
+mySubFolder = translatePath(path.join(myTmp, 'subs')).encode('utf-8')
+myName = myAddon.getAddonInfo('name')
+myLang = myAddon.getLocalizedString
+
+def getDomain():
+	try:
+		myDomain = str(get('https://pastebin.com/raw/1vbRPSGh').text)
```

Comment:
sorry this is not allowed. Either hardcode the url or move it to a setting. No pastebin shaddy links please.
If you control the domain manage the destination in your DNS configuration not in kodi.